### PR TITLE
Researcher profile integration

### PR DESCRIPTION
**Pull request by kjgarza**
## Description

Includes changes for the implementation of the researcher profile in Fabrica. It also includes the internal feedback changes.

- Changes in the landing page https://github.com/datacite/bracco/issues/292
- Changes in the Doi list https://github.com/datacite/bracco/issues/293
- TEST https://github.com/datacite/bracco/issues/291
- API integration https://github.com/datacite/bracco/issues/270
   - Using REST API

## List of General Components affected

changes focus in the User Router, a new metrics component, and changes i nthe DOIlist and DOI summary component

## Status
- [x] Ready for Review

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Non Functional Requirement
- [x] Follows the code style of this project.
- [x] Tests Cover Changes
- [ ] All new and existing tests passed.
- [ ] Documentation

### CODE REVIEW

File: app/components/doi-metrics-badge.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  metrics: [],
+  store: service(),
+  hasUsage: computed('views','downloads', function() {
+    return ((this.views + this.downloads) > 0) ?  true  : false;
+  }),
+  hasMetrics: computed('citations','views','downloads', function() {
+    return ((this.views + this.downloads + this.citations) > 0) ?  true  : false;
+  }),
+
+  init() {
```
Comment (mfenner) [inline app/components/doi-metrics-badge.js]: Why are you writing this, it is the default behaviour.

File: app/components/doi-metrics-badge.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+import { computed } from '@ember/object';
+
+
+export default Component.extend({
+  tagName: 'div',
```
Comment (mfenner) [inline app/components/doi-metrics-badge.js]: Why are you writing this, it is the default behaviour.

File: app/components/metrics-counter.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  label: '',
+  count: 0,
+  init() {
+    this._super();
+  },
+
+  didReceiveAttrs() {
+    this._super(...arguments);
+    if (this.data[`${this.label.toLowerCase()}s`] != null) {
+      this.set('count', this.data[`${this.label.toLowerCase()}s`]);
```
Comment (mfenner) [inline app/components/metrics-counter.js]: Please use `pluralize`helper.

File: app/routes/users/show.js [hunk 4]
```diff
@@ -1,37 +1,46 @@
...
-      return user;
-    }).catch(function(reason) {
-      console.debug(reason);
-
-      self.get('flashMessages').warning('Fabrica is currently unavailable due to a DataCite API problem. We apologize for the inconvenience and are working hard to restore the service. Please check back later or contact DataCite Support if you have a question.');
-      self.transitionTo('/');
+    // let variables = { id: `http://orcid.org/${params.user_id}` };
+
+    let parameters = {
+      'user-id': `http://orcid.org/${params.user_id}`,
```
Comment (mfenner) [inline app/routes/users/show.js]: The REST API does not use the URL form of the ID.

File: app/routes/users/show/dois.js [hunk 5]
```diff
@@ -7,17 +7,18 @@ export default Route.extend({
   can: service(),
 
   model(params) {
-    let user = this.modelFor('users/show');
+    let user = this.modelFor('users/show').user;
```
Comment (mfenner) [inline app/routes/users/show/dois.js]: Something is wrong here.

File: app/routes/users/show/dois.js [hunk 6]
```diff
@@ -7,17 +7,18 @@ export default Route.extend({
...
         number: params.page,
         size: params.size,
       },
       'user-id': user.get('id'),
+      'mix-in': 'metrics',
     });
 
     return hash({
-      user,
+      user: user.user,
```
Comment (mfenner) [inline app/routes/users/show/dois.js]: Something is wrong here, please fix.

File: app/routes/users/show/dois.js [hunk 7]
```diff
@@ -32,7 +33,7 @@ export default Route.extend({
   },
 
   afterModel() {
-    if (this.can.cannot('read user', this.modelFor('users/show'))) {
```
Comment (mfenner) [inline app/routes/users/show/dois.js]: Something is wrong here.

File: app/routes/users/show/settings.js [hunk 8]
```diff
@@ -6,7 +6,7 @@ export default Route.extend({
 
   model() {
     let self = this;
-    return this.store.findRecord('user', this.modelFor('users/show').get('id')).then(function(user) {
+    return this.store.findRecord('user', this.modelFor('users/show').user.get('id')).then(function(user) {
```
Comment (mfenner) [inline app/routes/users/show/settings.js]: Something is wrong here.

File: app/routes/users/show/settings.js [hunk 9]
```diff
@@ -17,7 +17,7 @@ export default Route.extend({
   },
 
   afterModel() {
-    if (this.can.cannot('read user', this.modelFor('users/show'))) {
+    if (this.can.cannot('read user', this.modelFor('users/show').user)) {
```
Comment (mfenner) [inline app/routes/users/show/settings.js]: Something is wrong here.

File: app/templates/components/doi-metrics-badge.hbs [hunk 10]
```diff
@@ -0,0 +1,21 @@
+<div style="display:flex;">
+  {{#if hasMetrics}}
+      {{#if citations}}
+        <i style="margin-right: 1.5em; margin-bottom: 4px; margin-top: 4px">{{fa-icon "quote-right"}} {{ format-numbers citations }} {{  pluralize citations 'Citation' without-count=true}}</i>
```
Comment (mfenner) [inline app/templates/components/doi-metrics-badge.hbs]: Please update the stylesheet instead of inline `style`.

File: app/templates/components/doi-metrics-badge.hbs [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+      {{else}}
+        <i style="margin-right: 1.5em; margin-bottom: 4px; margin-top: 4px">{{fa-icon "info-circle"}}  No citations were reported.</i>
+      {{/if}}
+      {{#if hasUsage }}
+        {{#if views}}
+          <i style="margin-right: 1.5em; margin-bottom: 4px; margin-top: 4px"> {{fa-icon "eye"}}  {{ format-numbers views }}  {{ pluralize views 'View' without-count=true }} </i>
```
Comment (mfenner) [inline app/templates/components/doi-metrics-badge.hbs]: Please update stylesheet.

File: app/templates/components/doi-metrics-badge.hbs [hunk 12]
```diff
@@ -0,0 +1,21 @@
+<div style="display:flex;">
+  {{#if hasMetrics}}
+      {{#if citations}}
+        <i style="margin-right: 1.5em; margin-bottom: 4px; margin-top: 4px">{{fa-icon "quote-right"}} {{ format-numbers citations }} {{  pluralize citations 'Citation' without-count=true}}</i>
+      {{else}}
+        <i style="margin-right: 1.5em; margin-bottom: 4px; margin-top: 4px">{{fa-icon "info-circle"}}  No citations were reported.</i>
```
Comment (mfenner) [inline app/templates/components/doi-metrics-badge.hbs]: Please update stylesheet.

File: app/templates/components/doi-metrics-badge.hbs [hunk 13]
```diff
@@ -0,0 +1,21 @@
...
+      {{/if}}
+      {{#if hasUsage }}
+        {{#if views}}
+          <i style="margin-right: 1.5em; margin-bottom: 4px; margin-top: 4px"> {{fa-icon "eye"}}  {{ format-numbers views }}  {{ pluralize views 'View' without-count=true }} </i>
+        {{/if}}
+        {{#if downloads}}
+          <i style="margin-right: 1.5em; margin-bottom: 4px; margin-top: 4px"> {{fa-icon "download"}}  {{ format-numbers downloads }}   {{ pluralize downloads 'Download' without-count=true }} </i>
```
Comment (mfenner) [inline app/templates/components/doi-metrics-badge.hbs]: Please update stylesheet.

File: app/templates/components/doi-metrics-badge.hbs [hunk 14]
```diff
@@ -0,0 +1,21 @@
...
+        {{/if}}
+      {{else}}
+        <i style="margin-right: 1.5em; margin-bottom: 4px; margin-top: 4px">{{fa-icon "info-circle"}}  No usage information was reported.</i>
```
Comment (mfenner) [inline app/templates/components/doi-metrics-badge.hbs]: Please update stylesheet.

File: app/templates/components/doi-metrics-badge.hbs [hunk 15]
```diff
@@ -0,0 +1,21 @@
...
+      {{/if}}
+  {{else}}
+    <i style="margin-right: 1.5em; margin-bottom: 4px; margin-top: 4px">{{fa-icon "info-circle"}}  No citations were reported. No usage information was reported.</i>
```
Comment (mfenner) [inline app/templates/components/doi-metrics-badge.hbs]: Please update stylesheet.

File: app/templates/users/show.hbs [hunk 16]
```diff
@@ -1,8 +1,8 @@
-{{#if (can "read user" model)}}
+{{#if (can "read user" model.user)}}
```
Comment (mfenner) [inline app/templates/users/show.hbs]: Something is wrong.

File: app/templates/users/show/index.hbs [hunk 17]
```diff
@@ -1,10 +1,19 @@
 <div class="content">
   <div class="row row-chart">
-    <BarChart @data={{model.doiCount}} @label="DOI" @model={{model}} @link="users.show.dois" @summarize={{true}} />
-    <DonutChart @data={{model.resourceTypeCount}} @label="DOI" @model={{model}} @link="users.show.dois" />
+    {{#if (can "update user")}}
```
Comment (mfenner) [inline app/templates/users/show/index.hbs]: Show the same chart (by publication year) to all users.

File: app/templates/users/show/index.hbs [hunk 18]
```diff
@@ -1,10 +1,19 @@
...
+      <BarChart @data={{model.user.doiCount}} @label="DOI" @model={{model.user}} @link="users.show.dois" @summarize={{true}} />
+    {{else}}
+      <BarChart @data={{model.user.doiCountByPublicationYear}} @label="DOI" @model={{model.user}} @link="users.show.dois" @summarize={{true}} />
+    {{/if}}
+    <DonutChart @data={{model.user.resourceTypeCount}} @label="DOI" @model={{model.user}} @link="users.show.dois" />
+  </div>
+  <div class="row row-chart">
+    <MetricsCounter @orcid={{model.user.id}} @data={{model.metrics}} @label='Citation' @model={{model}}  />
```
Comment (mfenner) [inline app/templates/users/show/index.hbs]: Something is wrong.

File: config/environment.js [hunk 19]
```diff
@@ -20,6 +20,9 @@ module.exports = function(environment) {
     sentry: {
       environment,
     },
+    apollo: {
```
Comment (mfenner) [inline config/environment.js]: Use an environmental variable instead of hard-coding the URL. And use the official GraphQL endpoint (`/graphql` instead of `/client-api/graphql`).

File: package.json [hunk 20]
```diff
@@ -148,7 +151,7 @@
     "ember-data-model-fragments": "^4.0.0",
     "ember-export-application-global": "^2.0.1",
     "ember-feature-flags": "^6.0.0",
-    "ember-fetch": "^7.0.0",
+    "ember-fetch": "^6.3.1",
```
Comment (mfenner) [inline package.json]: Why are you downgrading?

**Comment by kjgarza** [comment]
@mfenner 
> What made you decide to use GraphQL for users after all?

No, I'm calling the REST api. However I have left the Apollo client set up in. I can remove it completely, but I thought maybe moving forward we can start using it.


**The screenshots are in here :**
https://github.com/datacite/bracco/issues/292

**Comment by mfenner** [comment]
Please remove the Apollo client if you are not using GraphQL.

Please add a screenshot for the list view of DOIs.

**Comment by kjgarza** [comment]
Screenshot for DOI list
image

Screenshot for Landing page
image

File: package.json (hunk 20 above)
Comment (kjgarza) [reply package.json]: changed

File: app/templates/users/show/index.hbs (hunk 18 above)
Comment (kjgarza) [reply app/templates/users/show/index.hbs]: already changed

File: app/templates/users/show/index.hbs (hunk 17 above)
Comment (kjgarza) [reply app/templates/users/show/index.hbs]: already changed

File: app/templates/users/show.hbs (hunk 16 above)
Comment (kjgarza) [reply app/templates/users/show.hbs]: already changed

File: app/routes/users/show/dois.js (hunk 5 above)
Comment (kjgarza) [reply app/routes/users/show/dois.js]: already changed

File: app/routes/users/show.js (hunk 4 above)
Comment (kjgarza) [reply app/routes/users/show.js]: already changed

File: app/components/metrics-counter.js (hunk 3 above)
Comment (kjgarza) [reply app/components/metrics-counter.js]: already changed

File: app/components/doi-metrics-badge.js (hunk 1 above)
Comment (kjgarza) [reply app/components/doi-metrics-badge.js]: doi-metrics-badge.js  has been removed

File: app/components/doi-metrics-badge.js (hunk 2 above)
Comment (kjgarza) [reply app/components/doi-metrics-badge.js]: doi-metrics-badge.js  has been removed

File: app/abilities/provider.js [hunk 21]
```diff
@@ -31,16 +31,6 @@ export default Ability.extend({
         return false;
     }
   }),
-  canToken: computed('currentUser.role_id', 'model.provider_id', function() {
```
Comment (mfenner) [inline app/abilities/provider.js]: Please keep this code.

File: app/abilities/repository.js [hunk 22]
```diff
@@ -46,18 +46,6 @@ export default Ability.extend({
         return false;
     }
   }),
-  canToken: computed('currentUser.role_id', 'model.provider.id', function() {
```
Comment (mfenner) [inline app/abilities/repository.js]: Please keep this code.

File: app/components/doi-summary.js [hunk 23]
```diff
@@ -1,8 +1,33 @@
...
+  views: 0,
+  downloads: 0,
+  store: service(),
+  hasUsage: computed('views','downloads', function() {
+    return ((this.views + this.downloads) > 0) ?  true  : false;
+  }),
+  hasMetrics: computed('views','downloads','citations',function() {
+    return ((this.views + this.downloads + this.citations) > 0) ?  true  : false;
+  }),
+  isPerson: computed(function() {
```
Comment (mfenner) [inline app/components/doi-summary.js]: Can this also be done in the template?

File: app/components/doi-summary.js [hunk 24]
```diff
@@ -1,8 +1,33 @@
...
+    return this.attrs.link == 'users.show.dois';
+  }),
+  didReceiveAttrs() {
+    this._super(...arguments);
+
+    this.passMetrics();
+  },
+  passMetrics() {
```
Comment (mfenner) [inline app/components/doi-summary.js]: Can you make this easier by passing model.views, etc.? Why this new variable?

File: app/components/metrics-counter.js [hunk 25]
```diff
@@ -0,0 +1,20 @@
...
+  tagName: 'div',
+  classNames: [ 'col-lg-3', 'col-md-4' ],
+  data: null,
+  label: '',
+  count: 0,
+  init() {
+    this._super();
+  },
+
+  didReceiveAttrs() {
```
Comment (mfenner) [inline app/components/metrics-counter.js]: Why are you not doing this in the template, using the pluralize helper?

File: app/helpers/select-metrics.js [hunk 26]
```diff
@@ -0,0 +1,28 @@
+import { helper as buildHelper } from '@ember/component/helper';
+// import { htmlSafe } from '@ember/template';
+
+
+export function selectMetrics([ metrics, doi ]) {
```
Comment (mfenner) [inline app/helpers/select-metrics.js]: What is the function of this helper?

File: app/models/doi.js [hunk 27]
```diff
@@ -173,6 +173,11 @@ export default DS.Model.extend(Validations, {
   registered: DS.attr('date'),
   updated: DS.attr('date'),
   mode: DS.attr('string'),
+  meta: DS.attr(),
+  citations: DS.attr(),
```
Comment (mfenner) [inline app/models/doi.js]: Can you specify the attribute as DS.attr('number')?

File: app/models/provider.js [hunk 28]
```diff
@@ -126,7 +126,6 @@ export default DS.Model.extend(Validations, {
   name: DS.attr('string'),
   displayName: DS.attr('string'),
   symbol: DS.attr('string'),
-  globusUuid: DS.attr('string'),
```
Comment (mfenner) [inline app/models/provider.js]: please don't remove code.

File: app/models/repository.js [hunk 29]
```diff
@@ -115,7 +115,6 @@ export default DS.Model.extend(Validations, {
   name: DS.attr('string'),
   alternateName: DS.attr('string'),
   symbol: DS.attr('string'),
-  globusUuid: DS.attr('string'),
```
Comment (mfenner) [inline app/models/repository.js]: please don't remove code.

File: app/routes/users/show.js [hunk 30]
```diff
@@ -1,37 +1,43 @@
...
+        }).catch(function(reason) {
+          console.debug(reason);
+
+          self.get('flashMessages').warning('Fabrica is currently unavailable due to a DataCite API problem. We apologize for the inconvenience and are working hard to restore the service. Please check back later or contact DataCite Support if you have a question.');
+          self.transitionTo('/');
+        }),
+        self.store.query('doi', parameters).then(function(result) {
+          return result.meta;
+        }).catch(error => console.log(error)) ])
+        .then(function([ hashA, hashB ]) {
```
Comment (mfenner) [inline app/routes/users/show.js]: can you explain this line?

File: app/templates/components/doi-list.hbs [hunk 31]
```diff
@@ -240,7 +240,7 @@
       </div>
     </div>
   {{/if}}
-  {{#unless isPerson}}
+  {{#unless (and isPerson (cannot "update user"))}}
```
Comment (mfenner) [inline app/templates/components/doi-list.hbs]: avoid complex boolean (use if rather than unless).

File: app/templates/components/doi-list.hbs [hunk 32]
```diff
@@ -302,36 +302,38 @@
...
     </div>
   {{/if}}
-  {{#if (or model.meta.schemaVersions model.query.schema-version)}}
-    <div class="panel facets">
-      <div class="panel-body">
-        <h4>Schema Version</h4>
-        <ul>
-          {{#if model.meta.schemaVersions}}
-            {{#each model.meta.schemaVersions as |schemaVersion|}}
+  {{#unless (and isPerson (cannot "update user"))}}
```
Comment (mfenner) [inline app/templates/components/doi-list.hbs]: avoid complex boolean (see above).

File: app/templates/components/doi-summary.hbs [hunk 33]
```diff
@@ -30,8 +30,35 @@
...
+              <i class="metrics-icons"> {{fa-icon "eye"}}  {{ format-numbers views }}  {{ pluralize views 'View' without-count=true }} </i>
+            {{/if}}
+            {{#if downloads}}
+              <i class="metrics-icons"> {{fa-icon "download"}}  {{ format-numbers downloads }}   {{ pluralize downloads 'Download' without-count=true }} </i>
+            {{/if}}
+          {{else}}
+            <i class="metrics-icons">{{fa-icon "info-circle"}}  No usage information was reported.</i>
+          {{/if}}
+      {{else}}
+        <i>{{fa-icon "info-circle"}}  No citations were reported. No usage information was reported.</i>
```
Comment (mfenner) [inline app/templates/components/doi-summary.hbs]: Is there a way to remove some duplication in this code? And please look at whitespace.

File: app/templates/components/metrics-counter.hbs [hunk 34]
```diff
@@ -0,0 +1,15 @@
+<div class="panel panel-default">
+  <div class="panel-heading">
+    <h3 class="panel-title">
+      {{pluralize count label without-count=true}}
```
Comment (mfenner) [inline app/templates/components/metrics-counter.hbs]: don't you already pluralize in the component?

File: app/templates/providers/new.hbs [hunk 35]
```diff
@@ -27,13 +27,7 @@
 
           {{#if (can "create provider")}}
             <form.element @controlType="text" @id="salesforce-id" @label="Salesforce ID" @property="salesforceId" @required={{false}} @helpText="The 18 digit ID in Salesforce." />
-          {{/if}}
```
Comment (mfenner) [inline app/templates/providers/new.hbs]: don't remove code.

File: app/templates/providers/show/edit.hbs [hunk 36]
```diff
@@ -18,13 +18,7 @@
 
       {{#if (can "create provider")}}
         <form.element @controlType="text" @id="salesforce-id" @label="Salesforce ID" @property="salesforceId" @required={{false}} @helpText="The 18 digit ID in Salesforce." />
-      {{/if}}
-
-      {{#if (can "token provider")}}
```
Comment (mfenner) [inline app/templates/providers/show/edit.hbs]: don't remove code.

File: app/templates/providers/show/repositories/new.hbs [hunk 37]
```diff
@@ -14,10 +14,6 @@
         <form.element @controlType="text" @id="salesforce-id" @label="Salesforce ID" @property="salesforceId" @required={{false}} @helpText="The 18 digit ID in Salesforce." />
       {{/if}}
 
-      {{#if (can "token provider")}}
```
Comment (mfenner) [inline app/templates/providers/show/repositories/new.hbs]: don't remove code.

File: app/templates/providers/show/settings.hbs [hunk 38]
```diff
@@ -33,14 +33,11 @@
           <a href="{{model.rorId}}">{{model.rorId}}</a>
         {{/if}}
 
-        {{#if (and (can "create provider") model.salesforceId)}}
```
Comment (mfenner) [inline app/templates/providers/show/settings.hbs]: don't remove code.

File: app/templates/repositories/show/edit.hbs [hunk 39]
```diff
@@ -14,10 +14,6 @@
         <form.element @controlType="text" @id="salesforce-id" @label="Salesforce ID" @property="salesforceId" @required={{false}} @helpText="The 18 digit ID in Salesforce." />
       {{/if}}
 
-      {{#if (can "token provider")}}
```
Comment (mfenner) [inline app/templates/repositories/show/edit.hbs]: don't remove code.

File: app/templates/repositories/show/settings.hbs [hunk 40]
```diff
@@ -52,12 +52,7 @@
 
         {{#if (and (can "create provider") model.salesforceId)}}
           <h5>Salesforce ID</h5>
-          {{model.salesforceId}}
-        {{/if}}
-
-        {{#if (and (can "token repository") model.globusUuid)}}
```
Comment (mfenner) [inline app/templates/repositories/show/settings.hbs]: don't remove code.

File: app/templates/users/show/dois.hbs [hunk 41]
```diff
@@ -1 +1 @@
-<DoiList @model={{model.dois}} @link="users.show.dois" @searchable={{true}} />
+<DoiList @model={{model.dois}} @data={{model}} @link="users.show.dois" @searchable={{true}} />
```
Comment (mfenner) [inline app/templates/users/show/dois.hbs]: Why is this needed?

File: app/templates/users/show/index.hbs [hunk 42]
```diff
@@ -3,6 +3,12 @@
     <BarChart @data={{model.doiCount}} @label="DOI" @model={{model}} @link="users.show.dois" @summarize={{true}} />
     <DonutChart @data={{model.resourceTypeCount}} @label="DOI" @model={{model}} @link="users.show.dois" />
   </div>
+  <div class="row row-chart">
+
+    <MetricsCounter @orcid={{model.id}}  @label='Citation' @model={{model}}  />
```
Comment (mfenner) [inline app/templates/users/show/index.hbs]: why variable @orcid? This is the user-id.

File: config/environment.js (hunk 19 above)
Comment (mfenner) [inline config/environment.js]: are you using apollo? Don't use the URL pointing directly at client-api.

File: ember-cli-build.js [hunk 43]
```diff
@@ -34,7 +34,9 @@ module.exports = function(defaults) {
         content: (process.env.CDN_URL || 'https://assets.datacite.org') + '/stylesheets/doi.css?version=' + (pkg.version || '1.0'),
       },
     },
-
+    emberApolloClient: {
```
Comment (mfenner) [inline ember-cli-build.js]: are you using apollo?

File: fastboot-server.js [hunk 44]
```diff
@@ -6,7 +6,7 @@ if (typeof(PhusionPassenger) !== 'undefined') {
 
 const express = require('express');
 const compression = require('compression');
-const morgan  = require('morgan');
```
Comment (mfenner) [inline fastboot-server.js]: don't change production code not related to your pull request.

File: fastboot-server.js [hunk 45]
```diff
@@ -21,7 +21,7 @@ app.get('/assets/*', function(req, res) {
 });
 
 // logging
-app.use(morgan('combined'));
```
Comment (mfenner) [inline fastboot-server.js]: don't change production code unrelated to your pull request.

File: package.json [hunk 46]
```diff
@@ -173,14 +175,15 @@
     "eslint": "^6.8.0",
     "eslint-plugin-ember": "^7.7.2",
     "eslint-plugin-node": "^11.0.0",
+    "express-pino-logger": "^4.0.0",
```
Comment (mfenner) [inline package.json]: don't change production code unrelated to your pull request.

Comment (kjgarza) [reply package.json]: I think some of code we're change during the merge. I’ll correct that

File: fastboot-server.js (hunk 44 above)
Comment (kjgarza) [reply fastboot-server.js]: I think some of code we're change during the merge. I’ll correct that

File: config/environment.js (hunk 19 above)
Comment (kjgarza) [reply config/environment.js]: I missed to delete these

File: app/components/metrics-counter.js (hunk 25 above)
Comment (kjgarza) [reply app/components/metrics-counter.js]: Because it's not a label which is pluralized but a key value

File: app/helpers/select-metrics.js (hunk 26 above)
Comment (kjgarza) [reply app/helpers/select-metrics.js]: I will get ride of this one, we don't need it after the last reviw

File: app/routes/users/show.js (hunk 30 above)
Comment (kjgarza) [reply app/routes/users/show.js]: It's just a merge of the user data and the metrics data. We merge them to use them in the model hook together

File: app/abilities/repository.js (hunk 22 above)
Comment (kjgarza) [reply app/abilities/repository.js]: I think some of code we're change during the merge. I’ll correct that

File: app/components/doi-summary.js (hunk 23 above)
Comment (kjgarza) [reply app/components/doi-summary.js]: I'll look into it

File: app/templates/components/metrics-counter.hbs (hunk 34 above)
Comment (kjgarza) [reply app/templates/components/metrics-counter.hbs]: This is the laBel pluralization
